server/viame_server/serializers/viame.py

```diff
@@ -3,11 +3,49 @@
 """
 import csv
 import re
+from dataclasses import dataclass, asdict, field
+from typing import List, Dict, Tuple, Optional, Union, Any
 
 from girder.models.file import File
 
 
-def _deduceType(value):
+@dataclass
+class Feature:
+    """Feature represents a single detection in a track."""
+
+    frame: int = None
+    bounds: List[float] = None
+    head: Optional[Tuple[str, str]] = None
+    tail: Optional[Tuple[str, str]] = None
+    fishLength: Optional[float] = None
+    attributes: Optional[Dict[str, Union[bool, float, str]]] = None
+
+
+@dataclass
+class Track:
+    begin: int
+    end: int
+    key: int
```

Comment:
Let's keep this as `trackId` for consistency.   

Also, I'd like to make `trackId` a string and cast it to a string in the parser below.  It's number-ness is never used (for indexing, for example) so we might as well support other types of track identifiers, like UUIDs and things.

Reply:
Good point, I'm in favor of that.